Code.gs

```diff
@@ -130,20 +129,17 @@ function startSync(){
   if (onlyFutureEvents)
     startUpdateTime = new ICAL.Time.fromJSDate(new Date());
   
-  //Disable email notification if no mail adress is provided 
-  emailSummary = emailSummary && email != "";
-  
-  sourceCalendars = condenseCalendarMap(sourceCalendars);
-  for (var calendar of sourceCalendars){
+  var addedEvents = [];
+  var modifiedEvents = [];
+  var removedEvents = [];
+
+  for (var calendar of condenseCalendarMap(sourceCalendars)){
     //------------------------ Reset globals ------------------------
     calendarEvents = [];
     calendarEventsIds = [];
     icsEventsIds = [];
     calendarEventsMD5s = [];
     recurringEvents = [];
-    addedEvents = [];
-    modifiedEvents = [];
-    removedEvents = [];
```

Comment:
Does this change mean that `addedEvents`, etc is now relevant to all calendars and not calendar-specific?

Reply:
Oh, lower I think I see that you're just adding `condenseCalendarMap` to references instead

Reply:
In a nutshell, I removed the lines that modify the original arrays to prevent unintended changes or consequences in other locations. `addedEvents` was aggregated events from every calendar and that behavior hasn't changed. The difference is that the various events aggregators are not global and should be easier to pass around without causing bugs like #196.